packages/in_app_purchase/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -116,3 +138,33 @@ class BillingResponse {
   static const BillingResponse ITEM_ALREADY_OWNED = BillingResponse._(7);
   static const BillingResponse ITEM_NOT_OWNED = BillingResponse._(8);
 }
+
+/// Enum representing potential [SkuDetailsWrapper.type]s.
+///
+/// Wraps
+/// [`BillingClient.SkuType`](https://developer.android.com/reference/com/android/billingclient/api/BillingClient.SkuType)
+/// See the linked documentation for an explanation of the different constants.
+class SkuType {
```

Comment:
offline discussion: we're going to make this and BillingResponse an enum, but we'll leave this as-is in this PR as the following PR that adds json_serializable is going to do that anyway.